cpp/tests/io/comp/decomp_test.cpp

```diff
@@ -39,19 +39,19 @@ using cudf::device_span;
  */
 template <typename Decompressor>
 struct DecompressTest : public cudf::test::BaseFixture {
-  std::vector<uint8_t> vector_from_string(char const* str) const
+  [[nodiscard]] std::vector<uint8_t> vector_from_string(std::string const str) const
```

Comment:
Nit:  Why not `std::string const& str`?  It's not like `str` is being `std::move()`-ed.

Reply:
Good question. I can make this change.